src/Chapter08/Listing08.03.ImplementingAnInterface.cs

```diff
@@ -71,6 +72,33 @@ string[] IListable.ColumnValues
         }
         #endregion
 
+        protected new string Name
+        {
+            get
+            {
+                return FirstName + " " + LastName;
+            }
+            set
+            {
+                // Split the assigned value into 
+                // first and last names
+                string[] names;
+                names = value.Split(new char[] { ' ' });
```

Comment:
Did the original not combine the declaration and assignment?

Reply:
It did not. I lifted the Name property from a previous listing. Other properties were not included in the listing so I didn't have to make a book update. So I can change the property to an override above as well.